packages/terra-icon/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Fixed lint errors
+  
```

Comment:
Linter changes aren't relevant to consumers so we can skip adding this entry. If there is a build error, we can force merge it with a comment e.g:

https://github.com/cerner/terra-framework/pull/1666#issuecomment-1574204451